app/code/Magento/Customer/Test/Unit/Model/AccountManagementTest.php

```diff
@@ -995,8 +995,6 @@ protected function prepareInitiatePasswordReset($email, $templateIdentifier, $se
     {
         $websiteId = 1;
 
-        $dateTime = date(\Magento\Framework\Stdlib\DateTime::DATETIME_PHP_FORMAT);
```

Comment:
Instead of assertion removal please inject https://github.com/magento/magento2/blob/2.2-develop/lib/internal/Magento/Framework/Intl/DateTimeFactory.php into object under test and mock it properly.

Reply:
@orlangur Done